frontend/lib/identification/util/activate_card.dart

```diff
@@ -76,9 +79,21 @@ Future<bool> activateCard(
         await ActivationErrorDialog.showErrorDialog(context, t.identification.codeRevoked);
       }
       return false;
-    case Enum$ActivationState.failed:
+    case Enum$ActivationState.not_found:
+      if (context.mounted) {
```

Comment:
❓ do we really have to check this on each case?
I think we could add before switch statement

```
  if (!context.mounted) {
    return false;
  }
```